src/synth/fluid_synth.c

```diff
@@ -3897,19 +3895,15 @@ fluid_synth_set_reverb_on(fluid_synth_t* synth, int on)
  * @note Currently private to libfluidsynth.
  */
 int
-fluid_synth_set_reverb_preset(fluid_synth_t* synth, int num)
-{
-  int i = 0;
-  while (revmodel_preset[i].name != NULL) {
-    if (i == num) {
-      fluid_synth_set_reverb (synth, revmodel_preset[i].roomsize,
-                              revmodel_preset[i].damp, revmodel_preset[i].width,
-                              revmodel_preset[i].level);
-      return FLUID_OK;
-    }
-    i++;
-  }
-  return FLUID_FAILED;
+fluid_synth_set_reverb_preset(fluid_synth_t* synth, unsigned int num)
+{
+  if (num >= FLUID_N_ELEMENTS(revmodel_preset))
+    return FLUID_FAILED;
```

Comment:
Please use `fluid_return_val_if_fail()` for consistency.

Reply:
Done.

Reply:
Dont see it. Forgot to push?

Reply:
Excuse, I was waiting before doing the final push, for reading your comments...